pkg/reconcilers/k8s/deployment.go

```diff
@@ -266,7 +266,16 @@ func (r *K8sReconciler) generateAPIServerDeployment(namespace, dbName string, is
 			},
 		},
 	}
-	if isOCP {
+	if isMicroShift {
+		deployment.Spec.Template.Spec.SecurityContext = &v1.PodSecurityContext{
+			RunAsNonRoot: pointer.Bool(true),
+			RunAsUser:    pointer.Int64(10001),
```

Comment:
I thought we used 65534 in other fixes. 65534 is pretty standard, usually having username "nobody". AFAIK, 10001 is not so standardized.